subsys/bluetooth/shell/bt.c

```diff
@@ -1074,6 +1093,25 @@ static int cmd_scan_filter_set_addr(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int cmd_scan_filter_set_rssi(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err = 0;
+	long rssi = shell_strtol(argv[1], 10, &err);
```

Comment:
I'd prefer to split out the `shell_strtol()` call into the function body rather than the declarations, since it depends on another local variable and it's initialisation. Just a style thing, but I'd find it easier to read that way (i.e. keep the variable declaration block simple)